control/controlknobs/controlknobs.go

```diff
@@ -62,8 +62,9 @@ type Knobs struct {
 	// netfiltering, unless overridden by the user.
 	LinuxForceNfTables atomic.Bool
 
-	// SeamlessKeyRenewal is whether to enable the alpha functionality of
-	// renewing node keys without breaking connections.
+	// SeamlessKeyRenewal is whether to renew node keys without breaking connections.
+	// This is enabled by default in 1.90 and later, but we have a penalty flag
+	// in control that allows us to disable it remotely.
```

Comment:
// in the control plane server to ...

(we avoid internal jargon like just "control" in this public repo)

Even "penalty flag" is too jargon-y. Maybe just

"but we can remotely override it from the control plane if there's a problem"